docs/sources/features/datasources/cloudwatch.md

```diff
@@ -103,14 +101,26 @@ Here is a minimal policy example:
 
 ### AWS credentials
 
-If Auth Provider is `Credentials file`, Grafana tries to get credentials in the following order.
+There are three different authentication methods available. `AWS SDK Default` performs no custom configuration at all and instead uses the [default provider](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html) as specified by the AWS SDK for Go. This requires you to configure your AWS credentials separately, such as if you've [configured the CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html), if you're [running on an EC2 instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html), [in an ECS task](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html) or for a [Service Account in a Kubernetes cluster](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+
+`Credentials file` corresponds directly to the [SharedCredentialsProvider](https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/#SharedCredentialsProvider) provider in the Go SDK. In short, it will read the AWS shared credentials file and find the given profile. While `AWS SDK Default` will also find the shared credentials file, this option allows you to specify which profile to use without messing with environment variables. It doesn't have any implicit fallbacks to other credential providers, and will fail if using credentials from the credentials file doesn't work.
+
+`Access & secret key` corresponds to the [StaticProvider](https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/#StaticProvider) and will use the given access key ID and secret key to authenticate. This method doesn't have any fallbacks, and will fail if the provided key pair doesn't work.
+
+### Assuming a role
+
+The `Assume Role ARN` field allows you to specify which IAM role should be assumed, if any. When left blank, the provided credentials will be used directly and the associated role or user should have the required permissions. If this field is non-blank, on the other hand, the provided credentials will be used to perform an [sts:AssumeRole](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html) call.
```

Comment:
Can we make this active voice, instead of passive? For example:

In the `Assume Role ARN` field, specify the IAM role to assume. When left blank, the credentials provided are used directly, and the associated role or user has the required permissions. If this field is non-blank, the credentials provided are used to perform an [sts:AssumeRole](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html) call.

Reply:
@achatterjee-grafana Why is "The `Assume Role ARN` field allows you to specify which IAM role should be assumed, if any" passive voice? When I read the documentation style guide, the following is considered active: "Grafana displays the heatmap visualization."

Reply:
I shortened that particular line and the rest of the "original" content used passive voice which I corrected. Hope this helps.

Reply:
"In the Assume Role ARN field, specify the IAM role to assume" sounds to me as if you're supposed to specify a role. Generally you don't, so I don't agree that this is better language.

Reply:
I also changed the language in the meantime, so please make a new comment with regards to the current text.

Reply:
In general, we direct a user to do something. For example, specify some value in a particular field. In your example from the style guide, Grafana displays something. 

The field you mention does not do something by itself - the user specifies something in that field. That is why I suggested starting with Specify ..., but if you disagree, we can revisit the style later. I like the change you made to the later part of the sentence. Another suggestion: In the `Assume Role ARN` field, specify which IAM role should be assumed if any. 

If you do not agree, we will let this go for now.